### DISCUSSION TITLE
My student ID got rejected

### DISCUSSION THREAD

**Discussion by trishitchar** [community/community#27501] [GitHub Education]
My student ID photo got rejected every time
Please help me out

**Comment by MathiasReker** [comment]
You have to contact your school about this.

**Reply by bagraves** [reply]
Or maybe the screening process is awful.

**Reply by trishitchar** [reply]
![Uploading Screenshot_2022-07-28-21-51-33-172_com.android.chrome.jpg…]()

**Comment by Pramanik4** [comment]
Means you are conveying that you are applying for github student developer pack and it is getting rejected due to your student ID???

**Reply by trishitchar** [reply]
Yes that's happening

**Comment by trishitchar** [comment]
![Uploading Screenshot_2022-07-28-21-51-33-172_com.android.chrome.jpg…]()
